### DISCUSSION TITLE
Assets - Accessing an Original File - forbidden

### DISCUSSION THREAD

**Discussion by km37500** [directus/directus#4977] [Archive]
Hi,

I want to access my uploaded asset file als
example.com/assets/1ac73658-8b62-4dea-b6da-529fbc9d01a4
https://docs.directus.io/reference/api/assets/

but the respond is 

`
  "errors": [
    {
      "message": "You don't have permission to access this.",
      "extensions": {
        "code": "FORBIDDEN"
      }
    }
  ]
}`

Where can I edit the access rights for assets?

**Comment by Feuerhamster** [comment]
Go to **Settings > Roles & Permissions** .
Select the role you want to give permission to.

_Note that you must authenticate to the API with a user that have the appropriate role.
However, you can also give permissions to the "Public" role.
Then you don't need authentication, but anyone can access your files._

After you have selected the role, unfold "System collections" and set the read permission at "Directus Files".

![image](https://user-images.githubusercontent.com/38376566/114308792-4b830c80-9ae5-11eb-9b30-d47c133cdb98.png)

Then you should be able to access the files you want.
More information about permissions: https://docs.directus.io/guides/permissions/

**Reply by km37500** [reply]
thank you very much
